docs/duration.md

```diff
@@ -89,16 +89,12 @@ Any missing ones will be assumed to be 0.
 Any non-object value is converted to a string, which is expected to be in ISO 8601 format.
 
 The `overflow` option controls how out-of-range values are interpreted:
-- `constrain` (the default): Infinite values are clamped to `Number.MAX_VALUE` or `-Number.MAX_VALUE`.
-  Values higher than the next highest unit (for example, 90 minutes) are left as-is.
-- `balance`: Infinite values will cause the function to throw a `RangeError`.
-  Values higher than the next highest unit, are converted to be in-range by incrementing the next highest unit accordingly.
+- `constrain` (the default): Values higher than the next highest unit (for example, 90 minutes) are left as-is.
```

Comment:
I'm now finding it hard to figure out why this would be called "constrain".

Reply:
The rationale was that the values are "constrained" to have non-mixed signs, "constrain" is the default for other types so there's a symmetry there, and that there were no better suggestions. (I agree, it makes more sense for plus() and minus() where the values may actually have to be changed to meet the no-mixed-signs constraint.)

Reply:
Yep. Now that we have negative durations, we had to have the same options for both `plus` and `minus`. `'constrain'` was the only term we came up with that made the most sense both for the addition and the subtraction case.